streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignor.java

```diff
@@ -45,8 +45,8 @@
 
 public class HighAvailabilityTaskAssignor implements TaskAssignor {
     private static final Logger log = LoggerFactory.getLogger(HighAvailabilityTaskAssignor.class);
-    private static final int DEFAULT_STATEFUL_TRAFFIC_COST = 10;
-    private static final int DEFAULT_STATEFUL_NON_OVERLAP_COST = 1;
+    public static final int DEFAULT_STATEFUL_TRAFFIC_COST = 10;
```

Comment:
ditto here: let's use a different variable name for the different classes, too confusing if we're mixing them up by calling them outside of this class now. So I guess `DEFAULT_HIGH_AVAILABILITY_XXX_COST` (or maybe just `DEFAULT_HA_XXX_COST`?)